### ISSUE TITLE
[Adaptive Card iOS - InputsWithValidation.Json]: Voice Over is conveying the incorrect role as 'Button' for the 'Reply' link.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#178]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.5" control and select it.
4. Navigate on "V1.5" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "InputsWithValidation.json" control and double tap to activate it.
7. Perform right swipe and navigate to the card and observe the issue.

**Actual Result:**
While navigating to the 'Reply' link, voice over is conveying the incorrect role as 'Button'. It is announcing as 'Reply Button' and on activating it a new page is getting open.

**Expected Result:**
Screen Reader should convey the correct role information as 'Link' for the 'Reply' and it should announce as 'Reply Link'.

**User Impact:**
It will impact the screen reader user if the screen reader does convey the incorrect role for the element as user will get confused and treat the control as button instead of link.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/22d4b9b6-48d8-44d5-af81-16ad997a060e

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is still repro on the build 1.0 (2.4.0-beta.24.11.3.1 so we will reactivate this issue is next month first week.

![Image](https://github.com/user-attachments/assets/615db1f5-7e5b-4d9b-ab44-69b73747f480)

**Comment by MengliElmendorf137** [comment]
The issue is resolved. The trait of Reply is Link. You can also see it in your screenshot above.

![Image](https://github.com/user-attachments/assets/0b20ae7b-fbe3-4aa2-bbe3-0fb2f8c52043)
![Image](https://github.com/user-attachments/assets/a3f4166f-8b82-412b-9840-375ec2a3896a)
